controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/identifiers/DeploymentId.java

```diff
@@ -3,55 +3,52 @@
 
 import com.yahoo.config.provision.ZoneId;
 
+import java.util.Objects;
+
 /**
- * Application + zone. 
+ * ApplicationId x ZoneId.
```

Comment:
Should _x_ be _+_ ?

Reply:
Not if you ask me :p It is the [cartesian product](https://en.wikipedia.org/wiki/Cartesian_product) of the two.